CHANGELOG.md

```diff
@@ -3,8 +3,11 @@ Changes
 
 ## Unreleased
 
+This release contains a major refactoring.
```

Comment:
Document the X2Vec refactoring in the change log.

What changed from the user's perspective?